api-report/map.api.md

```diff
@@ -171,6 +174,10 @@ export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implem
     createSubDirectory(subdirName: string): IDirectory;
     delete(key: string): boolean;
     deleteSubDirectory(subdirName: string): boolean;
+    // (undocumented)
+    dispose(error?: Error): void;
```

Comment:
i would expect these to be optional. might need to make them explicitly optional in their signatures

Reply:
So, implementing Partial does implicitly imply that these could be optional. So the class implementing this interface, may/may not implement these apis and in both cases it should be fine.

Reply:
could just be an issue with the docs generation?